pulsar-io/core/src/main/java/org/apache/pulsar/io/core/SinkContext.java

```diff
@@ -42,4 +44,11 @@
      */
     String getSinkName();
 
+    /**
+     * Get subscription type
```

Comment:
We need to provide more information about what does subscription type means in `SinkContext`.